data/sql/updates/pending_db_world/rev_1623675094427536149.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1623675094427536149');
+
+-- Makes Theramore Combat Dummies boring again.
+UPDATE `creature_template` SET `mechanic_immune_mask` = 32, `flags_extra` = 262144, `ScriptName` = 'npc_training_dummy' WHERE (`entry` = 4952);
```

Comment:
https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits

Reply:
Cheers, rewritten to meet standard.

Reply:
Actually, just tested this and the combat dummies are dying if you hit them hard enough, so needs more work still.

Reply:
Was working with refreshed DB for other testing, so the above changes weren't included. Dummies running the new changes don't die, but they do take half-health damage, which the dummies I copied their settings from (Expert's Training Dummy in Org) do not, so still missing a difference somewhere.